modules/client/examples.md

```diff
@@ -884,9 +887,8 @@ const client = new TokenVotingClient(contextPlugin);
 
 const voteParams: ICanVoteParams = {
   address: "0x1234567890123456789012345678901234567890",
```

Comment:
Is `address` the address of the person to check whether he/she can vote?

I know it's not part of the PR per se, but this feels very ambiguous here. Could we update the type in all plugins, so that it's clear what it does?

I suggest `voterAddressOrEns` here and `approverAddressOrEns` on multisig (unless there's a better name)

Types, doc, examples and tests

Reply:
I have no clue. @josemarinas you changed these lines the last time so please answer the question

Reply:
Yes this is the wallet address that you want to check if hast the permission to vote. This means the `who`so yes maybe is not the most straightforward naming and for sure it should be changed at least to `addressOrEns`, the change to `voterXXX` or `approverXXX` wouñlld make sense because in other parts of the code we have a reference of whats the address for, for example `daoAddressOrEns` or `pluginAddress`.

So yeah, I'm happy with the change, let me know if you wanna include it in this PR. If not create a task to review all the taming follows this pattern 👍

Reply:
@josemarinas ok, then let's have this as a separate task not to make things bigger :+1: